module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,14 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <cctype>
+
+void removeVowels(std::vector<std::string>& words) {
+    std::vector<char> vowels{'a', 'e', 'i', 'o', 'u', 'y'};
```

Comment:
Remeber about upper-case letters.

Reply:
He is preparing strings by `std::tolower`

Reply:
Why not std::string?